packages/theme/src/cli/utilities/asset-checksum.ts

```diff
@@ -7,6 +7,8 @@ export async function checksum(root: string, path: string) {
 
   if (!content) return ''
 
+  if (Buffer.isBuffer(content)) return md5(content)
```

Comment:
⚠️ @karreiro this changes how images checksums are calculated. I noticed while testing push that the values were corrupt on Shopify. This fixes it, but has resuletd in some changed tests in `theme-fs.test.ts`